src/libs/clarinet.js

```diff
@@ -21,10 +21,10 @@ import { SAX_KEY, SAX_VALUE_OPEN, SAX_VALUE_CLOSE, FAIL_EVENT, STREAM_DATA, STRE
       FAIL_EVENT      
  */
 
-function clarinet(eventBus) {
+function clarinet(eventBus, options) {
   "use strict";
    
-  var 
+  var disableBufferCheck   = options && options.disableBufferCheck ? true : false,
```

Comment:
We might be able to set a default value for options to `{}` and then just check if `options.disableBufferCheck` is truthy. I think that the webpack build should transpile that

Reply:
Unfortunately we are not transpiling just yet. I could look at adding babel or typescript but for now we are just leveraging that webpack 3 natively supports modules.